Docs/source/usage/parameters.rst

```diff
@@ -2385,16 +2385,22 @@ Reduced Diagnostics
         * ``<reduced_diags_name>.reduction_type`` (`string`)
             The type of reduction to be performed. It must be either ``Maximum``, ``Minimum`` or
             ``Integral``.
-            ``Integral`` computes the spatial integral of the function defined in the parser by
-            summing its value on all grid points and multiplying the result by the volume of a
-            cell.
+            ``Integral`` computes the spatial surface or volume integral, depending on the choice
+            of the integration_type, of the function defined in the parser by summing its value on
+            all grid points and multiplying the result by the area or volume of a cell.
             Please be also aware that measuring maximum quantities might be very noisy in PIC
             simulations.
 
         The only output column is the reduced value.
 
         Note that the fields are averaged on the cell centers before the reduction is performed.
 
+        * ``<reduced_diags_name>.integration_type`` (`string`)
+           The type of integration to be performed. It must be either ``surface`` or ``volume``.
```

Comment:
```suggestion
        * ``<reduced_diags_name>.integration_type`` (`string`)
           The type of integration to be performed. It must be either ``surface`` or ``volume``. Note that for the surface integral, the user-defined parser for the plane should be less than one cell size in thickness in the direction of the surface normal. Also, the direction of the ``surface_normal`` must be specified for surface integration.
```

Reply:
In this case we should not write about the directionality of surface normal, it is just X or Y or Z.